crates/bevy_render/src/camera/projection.rs

```diff
@@ -244,6 +246,8 @@ impl CameraProjection for OrthographicProjection {
                 max_width,
                 max_height,
             } => {
+                // Compare Pixels of current width and maximal height and Pixels of maximal width with current height.
+                // Then use smaller (max_heigth when true) as what it referres to (height when true) and calculate rest so it can't get over maximum.
```

Comment:
```suggestion
                // Then use smaller (max_height when true) as what it referres to (height when true) and calculate rest so it can't get over maximum.
```